maestro/entities.py

```diff
@@ -851,6 +855,23 @@ def _parse_lifecycle(self, lifecycles):
 
         return checkers
 
+    def _parse_ulimits(self, ulimits):
+        """Parse ulimits"""
+        if ulimits is None:
+            return None
+        else:
+            result = []
+            for name, value in ulimits.items():
+                ulimit = {}
+                if isinstance(value, dict):
+                    ulimit.update({'name': name})
+                    ulimit.update(value)
+                elif isinstance(value, int):
+                    ulimit.update({'name': name})
+                    ulimit.update({'hard': value, 'soft': value})
```

Comment:
You should have a `else: continue` in here so you don't append an empty `ulimit` dictionary to the `result` list